src/ecosystemIndex.jsx

```diff
@@ -495,11 +495,11 @@ async function init() {
                     && (
                       <Route
                         exact
-                        path='/discovery/:studyUID'
+                        path='/discovery/:studyUID*'
```

Comment:
yes! the `*` is needed so it can treat strings of subpath that contains `/`s as a studyUID for whole. So this chagnes are need to be made to all the index pages we have for different portal bundles. This comment is applicable to all 3 comments below as well